echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -16,9 +20,6 @@
 import rx.functions.Action1;
 import rx.functions.Func1;
 
-import javax.annotation.PostConstruct;
```

Comment:
I think moving these up into the rest of the imports is not our usual convention.